### ISSUE TITLE
Audio only input / output streams

### ISSUE THREAD

**Issue by FilipStadler** [datarhei/restreamer#541]
I need a way to stream SRT audio only server with no video  signal 

- to make maybe Talkback to Larix Broadcaster or my own setup were SRT would be usefull - if I only make a audio stream it does not work  - it would be nice if a audio only solution was possible.

**Comment by ioppermann** [comment]
The Restreamer UI ist targeted to video content and audio-only output is currently only implemented in the Icecast publication service.

If you are only interested in using SRT, the use the built-in SRT server directly. You can build your own streamid. The format of the streamid is `[name],mode:[publish|request],token:[SRT token as defined in the settings], e.g.

`foobar,mode:publish,token:abc`, or in the full SRT URL: `srt://[ip:port]?mode=caller&transtype=live&streamid=foobar,mode:publish,token:abc` and for playing the same stream: `srt://[ip:port]?mode=caller&transtype=live&streamid=foobar,mode:request,token:abc`.

Please note: you can use the SRT server without creating a channel in the UI beforehand. Just use any name.

**Comment by itprotj** [comment]
I just started using Restreamer and I too was hoping to also take an audio-only input (In my case from a HLS stream). Maybe add this to the wishlist for this project?

**Comment by FilipStadler** [comment]
> I just started using Restreamer and I too was hoping to also take an audio-only input (In my case from a HLS stream). Maybe add this to the wishlist for this project?

I solution can bee to use the virtual test screen input low resolution but select srt audio input its a workaround 
in some cases usefull maybe... but maybe a static image would be nice.
